pydarn/plotting/rtp.py

```diff
@@ -14,8 +14,9 @@
 from matplotlib import dates, colors, cm, ticker
 from typing import List
 
-from pydarn import (dmap2dict, DmapArray, DmapScalar,
-                    rtp_exceptions, SuperDARNCpids, SuperDARNRadars,
+from pydarn import (dmap2dict, check_data_type, time2datetime,
+                    DmapArray, DmapScalar, rtp_exceptions,
```

Comment:
Does rtp_exceptions still exist?

Reply:
It does because there are some weird normalization errors you can get with it if you RTP plot a RAWACF file which I think is RTP specific.